kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/food2door/domain/producer/entity/producerinfo/HealthyShopInfo.java

```diff
@@ -0,0 +1,7 @@
+package com.kodilla.good.patterns.challenges.food2door.domain.producer.entity.producerinfo;
+
+final public class HealthyShopInfo extends ProducerInfo {
+    public HealthyShopInfo() {
+        this.name = "Healthy Shop";
+    }
```

Comment:
3 implementacje -Info sa zbedne. To bylo zaciemnia kod. Co wiecej HealthyInfo nie powinno byc uzyte w GlutenFreeShop wiec lamie to zasade Liskov'a